stm_workshop/lib/data/data.hpp

```diff
@@ -0,0 +1,131 @@
+#pragma once
```

Comment:
Again, not a big fan of copying this over. Having one central data structure is pretty bad design and was only the case due to threads being used. This year you can, and should, opt for sensible control flow. Not sure how difficult this is to fix fo the workshop but it's something to keep in mind for sure.

Reply:
will do boss